### PR TITLE
create intro page

### PR DESCRIPTION
**Pull request by salaos**

### CODE REVIEW

File: components/ChallengeList.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+              <>
+                <h2>
+                  <SmallRoundButton onClick={() => onDelete(challenge.id)}>
+                    ✖︎
+                  </SmallRoundButton>
+                </h2>
+                {showPicture && (
+                  <div>
+                    <img
+                      src="https://raw.githubusercontent.com/salaos/capstone-project/main/public/images/explosion.gif"
```
Comment (Satttoshi) [inline components/ChallengeList.js:73]: In a Next.js App it is better to use the integrated Image component from react

File: package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-router-dom": "^6.10.0",
+    "react-scroll": "^1.8.9",
```
Comment (Satttoshi) [inline package.json:20]: would you mind explaining me this lib?

**Review by Satttoshi — APPROVED** [review]
Small hint about Images.

Clean code, good structuring, very readable. Good Job!

LGTM :rocket:

File: package.json (hunk 2 above)
Comment (salaos) [reply package.json:20]: wanted to try a scroll effect but kicked it out - thanks, will take it out of the package json!

File: components/ChallengeList.js (hunk 1 above)
Comment (salaos) [reply components/ChallengeList.js:73]: True -thanks!

**Pull request merged**